### PR TITLE
added extra lines and instructions

### PR DESCRIPTION
**Pull request by ScriptingNoob**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -39,6 +39,11 @@
...
     100%{left: -20px}
 }
+
 p{
     text-align: center;
 }
+
+h1{
+    text-align: center;
+}
```
Comment (pjthepooh) [inline style.css:49]: add a new line at the end.

File: index.html [hunk 2]
```diff
@@ -5,6 +5,9 @@
     <title>Jump Game</title>
```
Comment (pjthepooh) [inline index.html:5]: How about changing the name to "Dinosaur Game"?

File: index.html [hunk 3]
```diff
@@ -5,6 +5,9 @@
...
     <link rel="stylesheet" href="style.css">
 </head>
+<h1>Welcome to the dinosaur game ripoff</h1>
```
Comment (pjthepooh) [inline index.html:8]: How about making it all cap, and get rid of ripoff.

**Review by pjthepooh — COMMENTED** [review]
Left some comments.

File: index.html (hunk 2 above)
Comment (ScriptingNoob) [reply index.html:5]: will do
sure

File: index.html (hunk 3 above)
Comment (ScriptingNoob) [reply index.html:8]: first half is good second half is a no

File: style.css (hunk 1 above)
Comment (ScriptingNoob) [reply style.css:49]: yes i agree

**Review by ScriptingNoob — COMMENTED** [review]
will do

**Review by pjthepooh — APPROVED** [review]

**Pull request merged**